docs/api/core/BufferGeometry.html

```diff
@@ -274,6 +274,12 @@ <h3>[method:BufferGeometry lookAt] ( [param:Vector3 vector] )</h3>
 		<h3>[method:null merge]( [param:BufferGeometry bufferGeometry], [param:Integer offset] )</h3>
 		<div>Merge in another BufferGeometry with an optional offset of where to start merging in.</div>
 
+		<h3>[method:BufferGeometry mergeGroups]()</h3>
+		<div>Merge groups with common materialIndex values to minimize the number of draw calls required to
+		render the geometery. This will reorder the elements of all attributes.
+		Indexed BufferGeometries or those using an [page:InterleavedBufferAttribute] are not supported.
```

Comment:
I think you should test for these cases, then, and issue a console warning.

Reply:
I had wondered about that.
Will do.